apps/address-service/domains/user/schema/User.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Generated by `createschema user.User 'name:Text;isAdmin:Checkbox;email:Text;password:Password;'`
+ */
+
+const { Text, Checkbox, Password } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@condo/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@condo/keystone/plugins')
+const { dvAndSender } = require('@address-service/domains/common/schema/plugins/dvAndSender')
+const access = require('@address-service/domains/user/access/User')
+
+
+const User = new GQLListSchema('User', {
```

Comment:
Is local user needed for this service? May it be replaced with condo user?

Reply:
Yes, the model is necessary. There will be no ability to register. The most probable way to create an instance will be OIDC auth.